webpack.common.config.ts

```diff
@@ -677,9 +677,16 @@ module.exports = {
       ]
     },
     {
-      test: /\.html$/,
+      test: {
+        include: /.html$/,
+        exclude: /directive\.html$/
+      },
       loader: 'underscore-template-loader'
     },
+    {
+      test: /directive\.html$/,
+      loader: 'html-loader'
+    },
```

Comment:
Added these to mark that files ending with `directive.html` should be loaded by `html-loader`.